### PR TITLE
unique name

### PR DESCRIPTION
**Pull request by abhisuri97**
- specified content type on request
- instance "name" is now the unique app name that heroku gives
- url_name is the herokufied input name

Question @sandlerben what was the regex match syntax for?

### CODE REVIEW

File: app/instances/views.py [hunk 1]
```diff
@@ -74,24 +74,23 @@ def launch():
...
-                    'name': herokuified_name
-                }
 
             resp = s.post(
                 'https://api.heroku.com/app-setups',
                 headers=headers,
                 json=data
             )
 
+            print(resp.json())
```
Comment (sandlerben) [inline app/instances/views.py]: can you remove this?

File: app/instances/views.py [hunk 2]
```diff
@@ -74,24 +74,23 @@ def launch():
...
+
             resp.raise_for_status()
 
             app_id = resp.json()['app']['id']
             app_setup_id = resp.json()['id']
+            app_name = resp.json()['app']['name']  # heroku app name
```
Comment (sandlerben) [inline app/instances/views.py]: ah so heroku auto-generates a name?

Comment (abhisuri97) [reply app/instances/views.py]: yep

**Comment by sandlerben** [comment]
you tested right? this looks good to me

**Comment by sandlerben** [comment]
the regex was to make sure the name wouldn't error on heroku (that's the regex for a valid heroku name).

**Pull request merged**